catalog/spatial/csw/spatial-csw-common/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/common/transaction/CswAction.java

```diff
@@ -1,36 +0,0 @@
-/**
```

Comment:
Was there any consideration to keeping a common CswAction interface that the other actions could extend?

Reply:
and was there not enough commonality to also have a parent class implementation?

Reply:
I considered creating a `CswAction` interface for the others to extend but I figured it wouldn't be worth it. It would only help with not duplicating the signatures for `getTypeName()` and `getHandle()` in the other 3 interfaces since the code will never reference the update, delete, and insert objects as `CswAction`s directly. Plus, we don't expect new csw actions to be added so I figured it would be simpler to just copy these methods in the `UpdateAction`, `InsertAction`, and `DeleteAction` interfaces. Thoughts @pklinef ?